edgelet/edgelet-kube/src/convert/to_k8s.rs

```diff
@@ -430,12 +463,22 @@ pub fn spec_to_role_binding(
     let mut annotations = BTreeMap::new();
     annotations.insert(EDGE_ORIGINAL_MODULEID.to_string(), spec.name().to_string());
 
+    let owner_reference = api_meta::OwnerReference {
```

Comment:
Once again, is it possible to DRY out this conversion code?

Reply:
Added this to my task list for further refactoring when we have figured out all the right  objects produced by iotedged and edgeAgent and the K8s object associations to each other.

Reply:
lets put it on a separate method in this module for now.